spec/config/routes_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe 'Routes' do
```

Comment:
Je suis confus: tu ne testes absolument pas l'app ici 😅

Je t'avoue que c'est un poil relou à tester les subdomains constraints, mais là ton test ne sert absolument à rien. Donc au choix:

1. On ne teste pas (par contre faut coder que le `/` renvoi le `pages#home` et non redirige sur `/home`, ce que tu fais ici)
2. On teste -> dans ce cas là faut configurer le host and stuff

Pour 2, ce qu'il faut faire:

* Ajouter dans `/etc/hosts` `entreprise.api.gouv.local dashboard.entreprise.api.gouv.local` à la ligne du localhost
* Faire ce helper dans le code des tests:

```ruby
  def configure_host(host)
    Capybara.current_session.driver.reset!
    default_url_options[:host] = host
    Capybara.always_include_port = true
    Capybara.app_host = "http://#{host}"
  end
```
* T'utilises le helper du dessus pour vérifier que `dashboard.entreprise.api.gouv.local` redirige sur le /login et que `entreprise.api.gouv.local` renvoi la vue `pages#home` (et non redirige) (en faisant une disjonction entre le domain `local/fr` si c'est dev/non-dev ofc)

Reply:
Bon j'ai fait une tentative pour le 2/ mais ça leak partout dans les tests pendant le CI c'est relou à mort. Je vais pour le 1/ du coup.